xsd/NeTEx_publication_timetable.xsd

```diff
@@ -1713,6 +1731,17 @@ Provides a general purose wrapper for NeTEx data content.</xsd:documentation>
 			<xsd:field xpath="@id"/>
 			<xsd:field xpath="@version"/>
 		</xsd:key>
+		<!-- =====AccessEquipment============================ -->
+		<xsd:keyref name="AccessEquipment_AnyKeyRef" refer="netex:AccessEquipment_AnyVersionedKey">
+			<xsd:selector xpath=".//netex:AccessEquipmentRef"/>
+			<xsd:field xpath="@ref"/>
+			<xsd:field xpath="@version"/>
+		</xsd:keyref>
+		<xsd:key name="AccessEquipment_AnyVersionedKey">
+			<xsd:selector xpath=".//netex:EntranceEquipment | .//netex:LiftEquipment | .//netex:LiftCallEquipment | .//netex:RampEquipment |.//netex:StaircaseEquipment   | .//netex:EscalatorEquipment  |  .//netex:TravelatorEquipment  | .//netex:RoughSurface | .//netex:PlaceLighting"/>
```

Comment:
```suggestion
			<xsd:selector xpath=".//netex:EntranceEquipment | .//netex:LiftEquipment | .//netex:LiftCallEquipment | .//netex:RampEquipment | .//netex:StaircaseEquipment | .//netex:EscalatorEquipment | .//netex:TravelatorEquipment  | .//netex:RoughSurface | .//netex:PlaceLighting"/>
```

Reply:
Interestingly it validated... Sorry I didn't remove the ending of the merge commit right.